app/controllers/api/graphql.php

```diff
@@ -58,7 +58,7 @@
     });
 
 App::post('/v1/graphql/mutation')
-    ->desc('GraphQL Endpoint')
+    ->desc('GraphQL endpoint')
```

Comment:
@abnegate Should this have a better desc?

Reply:
Yep we could add some more detail here, could be particularly useful since it ends up in SDK docblocks

Reply:
I've added a task for this. We'll address these separately to avoid scope creep.